web/src/main/java/com/graphhopper/countries/GermanySpatialRule.java

```diff
@@ -15,17 +15,18 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package com.graphhopper.routing.util.spatialrules.countries;
+package com.graphhopper.countries;
```

Comment:
Yes, let's keep the JSON parsing in the web module for now. But the implementations of the SpatialRule like Germany, Austria, or the SpatialRuleLookupArray, don't have anything to do with json and should live in the core. Don't you think?